tests/tests/test_mtls.py

```diff
@@ -123,32 +122,17 @@ def hsm_setup(self, pin, ssl_engine_id, device):
 
     def hsm_get_key_uri(self, pin, ssl_engine_id, device):
         pt11tool_output = device.run(
-            "export SOFTHSM2_CONF=/softhsm/softhsm2.conf; p11tool --login --provider=/usr/lib/softhsm/libsofthsm2.so --set-pin="
+            "p11tool --login --provider=/usr/lib/softhsm/libsofthsm2.so --set-pin="
             + pin
             + " --list-all-privkeys"
         ).rstrip("\n")
         key_uri = re.search(r"URL:\s(.*)", pt11tool_output).group(1)
         key_uri = key_uri + ";pin-value=" + pin
 
-        device.run("cp /etc/ssl/openssl.cnf /etc/ssl/openssl.cnf.backup")
-        device.run(
-            'echo -ne "[openssl_init]\nengines=engine_section\n\n[engine_section]\npkcs11 = pkcs11_section\n\n[pkcs11_section]\nengine_id = '
-            + ssl_engine_id
-            + '\nMODULE_PATH = /usr/lib/softhsm/libsofthsm2.so\ninit = 0\n" >> /etc/ssl/openssl.cnf'
-        )
-        device.run(
-            'sed -i.backup -e "/\\[Service\\]/ a Environment=SOFTHSM2_CONF=/softhsm/softhsm2.conf" /lib/systemd/system/%s.service'
-            % device.get_client_service_name()
-        )
         return key_uri
 
     def hsm_cleanup(self, device):
-        device.run(
-            "mv /lib/systemd/system/%s.service.backup /lib/systemd/system/%s.service || true"
-            % (device.get_client_service_name(), device.get_client_service_name())
-        )
-        device.run("rm -Rf /softhsm")
-        device.run("mv /etc/ssl/openssl.cnf.backup /etc/ssl/openssl.cnf || true")
+        device.run("rm -rf /var/lib/softhsm/tokens/*")
```

Comment:
I don't think we need to cleanup anything, really. We don't reuse the device for further tests.

Reply:
Hmm, yeh. I did this first time around. Agreed. Let's just keep it